cutter-plugin/R2RetDec.cpp

```diff
@@ -7,7 +7,7 @@
  */
 
 #include "R2RetDec.h"
-
+#include "../include/r2plugin/r2retdec.h"
```

Comment:
I think that this should be only

`#include "r2plugin/r2retdec.h"`

For this to work add the following to the `cutter-plugin/CMakeLists.txt` to [this list](https://github.com/avast/retdec-r2plugin/blob/1ba22472bf7718cf5b70130d249c95681ad71889/cutter-plugin/CMakeLists.txt#L19)
```
 target_link_libraries(r2retdec_cutter core_retdec)
```
This is needed anyways as we want the `core_retdec.cpp` to be linked to the cutter plugin too.

Also to build your branch with `-DBUILD_CUTTER_PLUGIN=on` I had to change the `GIT_TAG` to `master` so I think that this should be included in this PR too (in `deps/cutter/CMakeLists.txt`).

Reply:
Made these changes.